### PR TITLE
Added flake8-rst

### PR DESCRIPTION
**Pull request by FHaase**
Generated recipe via `conda skeleton pypi flake8-rst`

### CODE REVIEW

File: recipes/flake8-rst/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ec9c7b7a5475c0b3ce8aee4db951f33beffad68bdb79d99d45788d4aa083ce52
+
+build:
+  number: 0
```
Comment (synapticarbors) [inline recipes/flake8-rst/meta.yaml]: I think this should probably be `noarch: python`:

https://conda-forge.org/docs/meta.html#building-noarch-packages

File: recipes/flake8-rst/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  imports:
+    - flake8_rst
+  commands:
+    - flake8-rst --help
+
+about:
+  home: https://github.com/kataev/flake8-rst
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (synapticarbors) [inline recipes/flake8-rst/meta.yaml]: Please raise an issue about the `license_file` missing from the source distribution upstream and put a link to the issue as a comment in the `meta.yaml` as a reminder for future releases.

File: recipes/flake8-rst/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - flake8_rst
+  commands:
+    - flake8-rst --help
+
+about:
+  home: https://github.com/kataev/flake8-rst
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE                                 # LICENSE File currently missing from sdist.
```
[the remark this answers is not in the document]
Comment (FHaase) [reply recipes/flake8-rst/meta.yaml]: Thanks, I had no idea what the problem was with the linter.

**Comment by synapticarbors** [comment]
@FHaase -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**